src/richie/apps/search/indexers/organizations.py

```diff
@@ -47,68 +47,71 @@ class OrganizationsIndexer:
     display_fields = ["absolute_url", "logo", "title.*"]
 
     @classmethod
-    def get_es_documents(cls, index, action):
+    def get_es_documents(cls, index=None, action="index"):
```

Comment:
This is a nitpick but in courses and categories you put `get_es_document_for_course` first

Reply:
nice catch! I wanted to do it and forgot.